src/extension/app/components/modal/modal-container.js

```diff
@@ -85,7 +85,7 @@ export class ModalContainer extends LitElement {
 
   firstUpdated() {
     const dialogWrapper = this.shadowRoot.querySelector('sp-dialog-wrapper');
-    dialogWrapper.addEventListener(MODAL_EVENTS.CLOSE, () => {
+    dialogWrapper?.addEventListener(MODAL_EVENTS.CLOSE, () => {
```

Comment:
What happens if this listener isn't added?

Reply:
Yeah good call.. I switched to `queryAsync`. This should not happen now and optional chaining not required.